### PR TITLE
Document Hetzner installation

### PR DESCRIPTION
**Pull request by kvaps**
As part of https://github.com/aenix-io/cozystack/issues/79

### CODE REVIEW

File: content/en/docs/how-tos/hetzner.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: How to install Cozystack in Hetnzer
+linkTitle: Hetzner
+description: "How to install Cozystack in Hetzner"
+weight: 10
+---
+
+Boot server to recovery mode
```
Comment (themoriarti) [inline content/en/docs/how-tos/hetzner.md:8]: or can set Rescue mode

File: content/en/docs/how-tos/hetzner.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+```console
+# lsblk
+NAME        MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINTS
+nvme0n1     259:4    0 476.9G  0 disk
+nvme1n1     259:0    0 476.9G  0 disk
+```
+
+Download Talos Linux asset from the Cozystack's [releases page](https://github.com/aenix-io/cozystack/releases), and write it into disk:
+
+```bash
```
Comment (themoriarti) [inline content/en/docs/how-tos/hetzner.md:23]: cd /tmp

File: content/en/docs/how-tos/hetzner.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+You can find more comprehensive example under the [link](https://github.com/siderolabs/talos/blob/10f958cf41ec072209f8cb8724e6f89db24ca1b6/internal/app/machined/pkg/runtime/v1alpha1/platform/nocloud/testdata/metadata-v2.yaml)
+
+Umount cloud-init partition, sync changes, and reboot the server
+
+```bash
+umount /mnt
+sync
+reboot
```
Comment (themoriarti) [inline content/en/docs/how-tos/hetzner.md:79]: After reboot command server cannot restart, i try reboot by Reset in Hetzner robot panel.

When server start booting i see problem
hetzner

Comment (kvaps) [reply content/en/docs/how-tos/hetzner.md:79]: Okay I just added this warinng for now, let's solve it later
Screenshot 2024-04-16 at 18 37 37

**Pull request merged**